pmm/aws-staging-start.groovy

```diff
@@ -502,7 +502,7 @@ pipeline {
                             fi
                             [ -z "${CLIENTS}" ] && exit 0 || :
                             if [[ \$PMM_VERSION == pmm1 ]]; then
-                                bash /srv/pmm-qa/pmm-tests/pmm-framework.sh \
+                                sudo bash /srv/pmm-qa/pmm-tests/pmm-framework.sh \
```

Comment:
wouldn't it cause any further issues as we are going to run pmm-framework.sh under the root

Reply:
PMM-6834

Reply:
I don't see the need to run pmm-framework.sh with sudo, why we need this?